### ISSUE TITLE
The method `eth_getLogs` does not support `blockHash` param

### ISSUE THREAD

**Issue by bz888** [XinFinOrg/XDPoSChain#457]
When making a rpc query to `https://rpc.ankr.com/xdc`

```
{
    "jsonrpc": "2.0",
    "method": "eth_getLogs",
    "params": [
        {
            "blockHash": "0xdc480998d4335c61fe5d32802b455a6fc5fa8c08e1b55ebd4c9229adde15e99c"
        }
    ],
    "id": 1
}
```

it would return the log for the latest block instead of the blockHash provided in the param.
if this is expected result then https://www.ankr.com/docs/rpc-service/chains/chains-api/xdc/#parameters-27 should be updated.

**Comment by gzliudan** [comment]
Same with #179.

**Comment by gzliudan** [comment]
This issue is fixed by PR #486, #490, #491. We are testing them on devnet now.
